package.json

```diff
@@ -12,12 +12,12 @@
   },
   "scripts": {
     "build": "truffle build && yarn run networks-reset && yarn run ts",
-    "test": "truffle test {build/common/,}test/*js && yarn run test-ts",
-    "test-stablex": "yarn run test-js test/stablex/*",
-    "test-js": "truffle test",
+    "test": "truffle test && yarn run test-ts",
```

Comment:
Could you elaborate on the removal of these directories. I guess now we only run truffle test on `test/` and not the others which is fine because those other folders shouldn't ever contain smart contract tests anyway.

Reply:
Hehe, you elaborated it yourself. We just skip the process of building the js files from the .ts files. Hence, we need to run the tests no longer from the build folder.